src/Aspire.Hosting.Azure.AppContainers/api/Aspire.Hosting.Azure.AppContainers.cs

```diff
@@ -46,4 +48,28 @@ public AzureContainerAppCustomizationAnnotation(System.Action<AzureResourceInfra
 
         public System.Action<AzureResourceInfrastructure, global::Azure.Provisioning.AppContainers.ContainerApp> Configure { get { throw null; } }
     }
+}
+
+namespace Aspire.Hosting.Azure.AppContainers
+{
+    public partial class AzureContainerAppEnvironmentResource : AzureProvisioningResource
+    {
+        public AzureContainerAppEnvironmentResource(string name, System.Action<AzureResourceInfrastructure> configureInfrastructure) : base(default!, default!) { }
+
+        public BicepOutputReference ContainerAppDomain { get { throw null; } }
+
+        public BicepOutputReference ContainerAppEnvironmentId { get { throw null; } }
+
+        public BicepOutputReference ContainerAppEnvironmentName { get { throw null; } }
+
+        public BicepOutputReference ContainerRegistryManagedIdentityId { get { throw null; } }
+
+        public BicepOutputReference ContainerRegistryUrl { get { throw null; } }
+
+        public BicepOutputReference LogAnalyticsWorkspaceId { get { throw null; } }
+
+        public BicepOutputReference PrincipalId { get { throw null; } }
+
+        public BicepOutputReference PrincipalName { get { throw null; } }
```

Comment:
@eerhardt I could see us moving these out of this resource in the future and into separate resources. These are all coupled to ACR (except the LA workspace).  Should we keep these on this resource?

Reply:
[Remove AzureContainerAppEnvironmentResource BicepOutputReferences from public API (dotnet/aspire#8529)](https://github.com/dotnet/aspire/pull/8529)